test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-Json.Tests.ps1

```diff
@@ -142,6 +142,17 @@ Describe 'ConvertFrom-Json Unit Tests' -tags "CI" {
         $out[1] | Should -Be $null
         $out[2] | Should -Be 2
     }
+
+    It 'Order is preserved for a hashtable' {
```

Comment:
I am worried that this test will fail if the formatter is modified at some point to provide a slightly different output.

Reply:
We already have a bunch of tests like this for `Format-Table` and agree that if we change the format all of those tests need to be updated.